polylabel.js

```diff
@@ -86,14 +94,22 @@ function compareMax(a, b) {
     return b.max - a.max;
 }
 
-function Cell(x, y, h, polygon) {
+function Cell(x, y, h, polygon, centroidCell) {
     this.x = x; // cell center x
     this.y = y; // cell center y
     this.h = h; // half the cell size
     this.d = pointToPolygonDist(x, y, polygon); // distance from cell center to polygon
+    this.distanceToCentroid = centroidCell ? pointToPointDist(this, centroidCell) : 0;
```

Comment:
Note: The only case where `centroidCell` is not provided is when generating the centroid cell itself, so distance should be 0.